examples/LoRaDuplex/LoRaDuplex.ino

```diff
@@ -0,0 +1,101 @@
+/*
+  LoRa Duplex communication
+
+  Sends a message every half second, and polls continually
+  for new incoming messages. Implements a one-byte addressing scheme,
+  with 0xFF as the broadcast address.
+
+  Uses readString() from Stream class to read payload. The Stream class'
+  timeout may affect other functuons, like the radio's callback. For an 
+
+  created 28 April 2017
+  by Tom Igoe
+*/
+#include <SPI.h>              // include libraries
+#include <LoRa.h>
+const int csPin = 7;          // LoRa radio chip select
+const int resetPin = 6;       // LoRa radio reset
+const int irqPin = 1;         // change for your board; must be a hardware interrupt pin
+
+String outgoing;              // outgoing message
+byte msgCount = 0;            // count of outgoing messages
+byte localAddress = 0xBB;     // address of this device
+byte destination = 0xFF;      // destination to send to
+long lastSendTime = 0;        // last send time
+int interval = 2000;          // interval between sends
+
+void setup() {
+  while (!Serial);
+  Serial.begin(9600);                   // initialize serial
+  Serial.println("LoRa Duplex");
+  LoRa.setPins(csPin, resetPin, irqPin);// set CS, reset, IRQ pin
+
+  if (!LoRa.begin(915E6)) {             // initialize ratio at 915Mhz
+    Serial.println("LoRa init failed. Check your connections.");
+    while (true);                       // if failed, do nothing
+  }
+
+   LoRa.setTimeout(10);                  // set Stream timeout of 10ms
```

Comment:
My experience with the Stream timeouts has been that timeout(0) tends to miss bytes on long strings, so I default to 10. But since the packet is buffered, it might be alright.